### DISCUSSION TITLE
Automatic resubmission of failed job

### DISCUSSION THREAD

**Discussion by FA-VS** [DIRACGrid/DIRAC#7229] [Q&A] [answered]
Dear developers,

When first submitting a job, is there a way to tell Dirac to automatically resubmit it up to n times if it fails? Ideally I would like to do it through the API, but I am happy with a CLI/JDL solution too. From looking online, the solutions I found seem to require a "manual" request after the first job fails.

Thank you for your time,

Francisco

**Comment by fstagni** [comment, marked as answer]
Hi, this feature does not exist in standard DIRAC WMS, and it is not exposed through the API. The DIRAC "transformation system", which is used for running productions, instead add this and other features.

**Comment by FA-VS** [comment]
Ok, thank you very much for the quick answer!
